code/modules/mob/mob_verbs.dm

```diff
@@ -19,37 +19,36 @@
 		next_move = world.time + 2
 	return
 
-/mob/verb/point_to(atom/A in view())
+/mob/verb/point_to(atom/A in view(client.view + client.get_offset(), loc))
```

Comment:
That allows people to receive "pointed at" messages of far flung things they shouldn't see or arrow points from seemengly nowhere, no?

Reply:
If their offset is 3 in one direction, it will allow them to point 3 further in all directions, sadly there's no better way to do this.

Reply:
Still potentially unhelpful and may also work through cameras. The `get_offset()` proc is good though.

Reply:
Client.view is just the amount of tiles a client sees from their mob.

Reply:
Umh, thanks for cleaing it out.